### PR TITLE
[WIP] Data lecture

### PR DESCRIPTION
**Pull request by RazvanN7**
Adds the first exercice (sources + text) and a bit of the second exercice. This PR is supposed to add the entire Data chapter, each commit implementing a specific feature.

### CODE REVIEW

**Comment by RazvanN7** [comment]
To whom it may concern, currently you should just look at the `lecture` directory. The lab still needs some shaping.

File: content/chapters/data/lecture/slides.md [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+
+----
+## OS Interface To Virtual Memory
+
+* /proc/$pid/mem - access to virtual memory
+    - demo/proc_mem/
+* /proc/$pid/page_map - acces to page mappings
+    - demo/proc_pagemap/
+* /dev/mem - access to physical memory
+    - demo/proc_pagemap
```
Comment (razvand) [inline content/chapters/data/lecture/slides.md:409]: This appears twice. Is it correct?

**Review by razvand — CHANGES_REQUESTED** [review]
Add Makefile for slides (using `reveal-md`). See example from [the compute chapter](https://github.com/teodutu/operating-systems-oer/blob/compute-lecture/content/chapters/compute/lecture/Makefile).

File: content/chapters/data/lecture/slides.md (hunk 1 above)
Comment (RazvanN7) [reply content/chapters/data/lecture/slides.md]: Yes.

**Comment by RazvanN7** [comment]
@razvand I added the Makefile, turned all images into svg and added quizzes. All that I need to do now is to split the course into subfiles.

File: content/chapters/data/lecture/slides.md [hunk 2]
```diff
@@ -0,0 +1,425 @@
...
+* Python - performance for security, ease of use
+* D - can switch between both worlds
+
+----
+### Example
+
+* Python
+    - `demo/variables/d_vars.d`
+* D
+    - `demo/variables/python_vars.d`
```
Comment (alexandruradovici) [inline content/chapters/data/lecture/slides.md:123]:
```suggestion
* D
    - `demo/variables/d_vars.d`
* Python
    - `demo/variables/python_vars.d`
```

**Pull request merged**